NETCORE/src/Microsoft.ApplicationInsights.WorkerService/ApplicationInsightsExtensions.cs

```diff
@@ -21,6 +21,7 @@ public static partial class ApplicationInsightsExtensions
         /// <param name="services">The <see cref="IServiceCollection"/> instance.</param>
         /// <param name="instrumentationKey">Instrumentation key to use for telemetry.</param>
         /// <returns>The <see cref="IServiceCollection"/>.</returns>
+        [Obsolete("InstrumentationKey based global ingestion is being deprecated. Use an AddApplicationInsightsTelemetryWorkerService() overload to set ApplicationInsightsServiceOptions.ConnectionString. See https://github.com/microsoft/ApplicationInsights-dotnet/issues/2560 for more details.")]
```

Comment:
the message is not very clear to me.
There is simply no other overload which takes ConnectionString. So suggesting to use an overload without being very clear which overload we are talking about can be confusing.

Reply:
I reviewed all the messages and added additional details.

I reworded this specific message to say `Use an AddApplicationInsightsTelemetryWorkerService() overload to set ApplicationInsightsServiceOptions.ConnectionString via Action<ApplicationInsightsServiceOptions>`